pycbc/workflow/grb_utils.py

```diff
@@ -566,6 +575,15 @@ def make_pygrb_plot(workflow, exec_name, out_dir,
             if workflow.cp.has_option_tags(exec_name, log_flag,
                                            tags=[subsection]):
                 node.add_opt('--'+log_flag)
+    elif exec_name == 'pycbc_plot_bank_corner':
+        node.add_opt('--no-suptitle')
+        if inj_file:
+            node.add_opt('--title', f'\"{tags[0]} injections\"')
+            params = workflow.cp.get_opt_tags(exec_name, 'parameters', ['injs'])
+        else:
+            node.add_opt('--title', f'\"Template bank\"')
+            params = workflow.cp.get_opt_tags(exec_name, 'parameters', ['bank'])
+        node.add_opt('--parameters', params)
```

Comment:
I do not think you need to handle the parameter list explicitly here. Any option given in the executable's section via the config file will be passed straight as a command line option by the workflow base code.

Reply:
It is needed for the subsections (i.e. `pycbc_plot_bank_corner_NSBH`). And we need to pass the parameters separately because injections and bank plotting have different parameters.

Reply:
As far as I am aware, the functionality you describe comes "for free" already. In the config file you will have two sections
```
[plot_corner-bank]
parameters = mass1 mass2 spin1z spin2z
title = "Template bank"
no-suptitle =

[plot_corner-bnsinj]
parameters = mass1 mass2 distance inclination ra dec
title = "Injections"
no-suptitle =
```
When you instantiate `PlotExecutable` and call `create_node()` above, this should *automatically* pick up the right section and all the arguments therein. I do not think you need to explicitly call `add_opt()`.